docs/doc-requirements.txt

```diff
@@ -1,3 +1,5 @@
+# Note that there is also a Pipfile for this project if you are updating this 
+# file do not forget to update thr Pipfile accordingly
```

Comment:
```suggestion
# file do not forget to update the Pipfile accordingly
```

Reply:
ohhh right I forgot about the "suggest" thing :-)

Reply:
best new feature since the invention of coffee!